src/bridge/cockpitinternalmetrics.c

```diff
@@ -81,8 +81,6 @@ static MetricDescription metric_descriptions[] = {
   { "disk.all.read",    "bytes", "counter", FALSE, DISK_SAMPLER },
   { "disk.all.written", "bytes", "counter", FALSE, DISK_SAMPLER },
 
-  { "network.all.rx",       "bytes", "counter", FALSE, NETWORK_SAMPLER },
```

Comment:
I think we need to leave these, so that old shells can work the same as they work today with the new bridge.